paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerModule.kt

```diff
@@ -57,6 +57,16 @@ internal object FlowControllerModule {
         linkStore,
     )
 
+    @Provides
+    @Singleton
+    fun provideFlowControllerConfirmationHandler(
```

Comment:
Normally you'd have a binds method, and put the `@Inject` annotation on the `DefaultFlowControllerConfirmationHandler` constructor.